_data/versions.yml

```diff
@@ -1,4 +1,57 @@
 v2.6:
+- title: v2.6.6
+  note: |
+    17 January 2018
+ 
+    - Updates the node startup logging format to be consistent with other Calico logs. 
+
+    - Updates Typha and Felix to send logs to stdout instead of stderr.
+
+    - Resolves an issue when upgrading from old versions of Calico where the `k8s-policy-no-match` network
```

Comment:
In the past, we sometimes include links to the repos where the fix was made, I don't know if you think we should add those to these....it could be nice if time permits.